lib/vsc/utils/generaloption.py

```diff
@@ -63,10 +65,13 @@ def set_columns(cols=None):
         return
 
     if cols is None:
-        stty = '/usr/bin/stty'
-        if os.path.exists(stty):
+        if is_py_ver(3, 3):
```

Comment:
Is that "at least"?

Reply:
`>=`

Reply:
well actually is `>= 3.3` and `< 4`

Reply:
Since they are making heavy changes in 3.9 (afaik), would that comparison become a potential problem?

Reply:
Who knows, but we'll deal with that when we start using Python 3.9.

For now, we're only testing with Python 3.6, as that's the Python we'll be using on RHEL8.

We can easily extend to additional Python versions in Travis/Jenkins, and deal with failing tests.

For now, let's try and keep things rolling with Python 3.6...

Also, `shutil.get_terminal_size` is still there in 3.9, see https://docs.python.org/3.9/library/shutil.html#shutil.get_terminal_size